src/components/DevDummyLogin/index.js

```diff
@@ -192,6 +207,8 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
     if (savedPasswordText) setPasswordText(savedPasswordText);
   }, []);
 
+  console.log(dummyInput);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive information like user input to the console can pose security risks and should be avoided in production code. If this log is for debugging purposes, make sure to remove it before merging the code.
```diff
-  console.log(dummyInput);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->